README.md

```diff
@@ -1,8 +1,6 @@
 SonarQube Scanner for Jenkins
 =============================
 
-[![Build Status](https://travis-ci.org/SonarSource/sonar-scanner-jenkins.svg?branch=master)](https://travis-ci.org/SonarSource/sonar-scanner-jenkins) [![Quality Gate](https://next.sonarqube.com/sonarqube/api/project_badges/measure?project=org.jenkins-ci.plugins%3Asonar&metric=alert_status)](https://next.sonarqube.com/sonarqube/dashboard?id=org.jenkins-ci.plugins%3Asonar)

```

Comment:
travis is no longer used and the sonarqube badge seems to time out for me 9 times out of 10. (can restore that but it looks quite bad to view the README and have it failing